app/services/add_newest_time_entries.rb

```diff
@@ -0,0 +1,50 @@
+class AddNewestTimeEntries
```

Comment:
Can you explain me why you call services with a verb? You learned it this way, right? But what's the reason? I'm used to names like `TimeEntriesUpdater`.